bin/lintr

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env Rscript
+files <- commandArgs(trailing = TRUE)
+
+if (git2r::in_repository()) {
+  git_status <- git2r::status(staged = FALSE)
+  if (any(grepl("\\.lintr", git_status))) {
```

Comment:
I think we could limit this to the `.lintr` file in the current working directory and flatten a potentially nested list (e.g. git_status$unstaged$modified`) first.
```suggestion
  if (any(grepl(".lintr", unlist(git_status), fixed = TRUE))) {
```